templates/base/footer.tmpl

```diff
@@ -11,6 +11,8 @@
 
 	{{template "custom/body_outer_post" .}}
 
+	<noscript>{{template "base/footer_noscript" .}}</noscript>
```

Comment:
I think it would be best to have the `<noscript>` inside the new template instead of here.

Reply:
Yeah, as my gut feeling said, this renders an additional separator for everyone that does use JavaScript.

![image](https://github.com/go-gitea/gitea/assets/30193966/c4c73a56-2d3b-4efc-8438-66e479d807d4)

Reply:
I meant not inside the footer, but inside the new template.
I don't see why this should change anything?